README.md

```diff
@@ -97,14 +97,28 @@ ms-app://s-1-15-2-2123189467-1366327299-2057240504-936110431-2588729968-14545362
 
 #### Register the TodoListClient app
 
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
-1. Enter a friendly name for the application, for example 'TodoListClient-StoreApp' and select 'Native' as the *Application Type*.
-1. For the *Redirect URI*, enter value that you obtained during the previous step with the debugger.
-1. Click **Create** to create the application.
-1. In the succeeding page, Find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. Then click on **Settings**, and choose **Properties**.
-1. Configure Permissions for your application. To that extent, in the Settings menu, choose the 'Required permissions' section and then,
-   click on **Add**, then **Select an API**, and type `TodoListService-StoreApp` in the textbox. Then, click on  **Select Permissions** and select **Access 'TodoListService-StoreApp'**.
+1. In the **Azure Active Directory** pane, click on **[App registrations](https://go.microsoft.com/fwlink/?linkid=2083908)** and choose **New registration**.
+1. Enter a meaningful name for the application, for example 'TodoListClient-StoreApp'
+1. Leave **Supported account types** on the default setting of **Accounts in this organizational directory only**.
+1. For the *Redirect URI*, enter value that you obtained during the previous step with the debugger and select "Public client (mobile & desktop)" from the dropdown.
+1. Click **Register** to create the application.
+1. In the application's **Overview** page, find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. In the list of pages for the app, select **Expose an API**.

```

Comment:
@kjyam98 : did you try to run the sample?
This all section lines 106-121 really goes with the TodoListService, not the client.